Sources/tart/OCI/Layerizer/DiskV2.swift

```diff
@@ -13,25 +13,22 @@ class DiskV2: Disk {
     let mappedDisk = try Data(contentsOf: diskURL, options: [.alwaysMapped])
 
     // Compress the disk file as multiple individually decompressible streams,
-    // each equal ``Self.layerLimitBytes`` bytes or slightly larger due to the
-    // internal compressor's buffer
-    var offset: UInt64 = 0
-
-    while let (compressedData, uncompressedSize, uncompressedDigest) = try compressNextLayerOfLimitBytesOrMore(mappedDisk: mappedDisk, offset: offset) {
-      offset += uncompressedSize
+    // each equal ``Self.layerLimitBytes`` bytes or less due to LZ4 compression
+    for data in mappedDisk.chunks(ofCount: layerLimitBytes) {
```

Comment:
Should we put `layerLimitBytes` in the manifest in case it's gonna change at some point?

Reply:
I think it's already there by the virtue of passing the `UInt64(data.count)` as `uncompressedSize` argument: https://github.com/cirruslabs/tart/blob/13d5ddb4a4d79c0d7babfb24c23436834c1981d2/Sources/tart/OCI/Manifest.swift#L94-L96